controllers/activemqartemis_reconciler.go

```diff
@@ -2495,6 +2536,21 @@ func (r *ActiveMQArtemisReconcilerImpl) createExtraConfigmapsAndSecretsVolumeMou
 					}
 				}
 			}
+			if strings.HasSuffix(secret, brokerPropsSuffix) {
+				propsData, ok := bpBrokerPropsData[secret]
```

Comment:
The reading of the keys of the secrets with brokerPropsSuffix could be moved here to avoid to keep in memory a map with all secrets.

Reply:
+1